cmd/exit_sign.go

```diff
@@ -126,60 +133,119 @@ func runSignPartialExit(ctx context.Context, config exitConfig) error {
 		log.Info(ctx, "Both public key and index are specified, beacon node won't be checked for validator existence/liveness")
 	}
 
+	var exitBlobs []obolapi.ExitBlob
+	if config.All {
+		exitBlobs, err = signAllValidatorsExits(ctx, config, eth2Cl, shares)
+		if err != nil {
+			return errors.Wrap(err, "could not sign exits for all validators")
+		}
+	} else {
+		exitBlobs, err = signSingleValidatorExit(ctx, config, eth2Cl, shares)
```

Comment:
Can we use signAllValidatorsExits here, and instead of passing all shares we pass a slice with the single share we're interested in?

Reply:
Yes, I've been thinking about that, but when we are signing for single validator we are checking the config if the user passed pubkey or index or both, in exit all scenario we don't have none of those. We can still use the same function for both cases, but I will have to add some flags and checks which to determine if we have to check the config or not. It will make the code less readable imo.